### ISSUE TITLE
Accessibility: Invalid aria attribute on carousel

### ISSUE THREAD

**Issue by Viale-Stephane** [ng-bootstrap/ng-bootstrap#4416]
### Bug description:
Hello, when running an accessibility analysis tool (axe chrome extension) on the carousel demo page an issue occur.
The attribute aria-activedescendant can only be used on elements with a certain role.
The [documentation](https://www.w3.org/TR/wai-aria-1.2/#managingfocus_useragents) says it should have one of the following role: [combobox](https://www.w3.org/TR/wai-aria-1.2/#combobox), [textbox](https://www.w3.org/TR/wai-aria-1.2/#textbox) or [searchbox](https://www.w3.org/TR/wai-aria-1.2/#searchbox).

### Step that reproduces the issue:

- access the demo page: https://ng-bootstrap.github.io/#/components/carousel/examples
- run axe accessibility analysis
- issue is reported
![image](https://user-images.githubusercontent.com/34646444/200776562-6981052b-771e-4c37-9b4a-87deddaa204a.png)

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: (your latest demo page, 14.1.0)

ng-bootstrap: (your latest demo page, 13.1.0)

Bootstrap: (your latest demo page, 5.2.0)

Thanks !
